.github/workflows/dev-builds.yml

```diff
@@ -31,13 +29,6 @@ jobs:
 
   build_mac_version:
     runs-on: macos-latest
-    env:
```

Comment:
This one is used for PR builds. QAs need to test a signed version of the app.

Reply:
Will fix, thanks for validating this, I checked the wrong 'scripts'.

Reply:
Added CSC back without GH_TOKEN. Let's wait for the CI to pass.